app/controllers/observations_controller.rb

```diff
@@ -29,4 +29,35 @@ class ObservationsController < ApplicationController
     # an observation from a species_list, but I can't figure out how.
     :edit
   ]
+
+  # NOTE: These ivars need to be ivars of ObservationController
+  # Defining them in index.rb does not work
+  @index_subaction_param_keys = [
+    :advanced_search, # Searches come 1st because they may have the other params
+    :pattern,
+    :look_alikes,
+    :related_taxa,
+    :name,
+    :user,
+    :location,
+    :where,
+    :project,
+    :by,
+    :q,
+    :id
+  ].freeze
+
+  @index_subaction_dispatch_table = {
+    pattern: :observation_search,
+    look_alikes: :observations_of_look_alikes,
```

Comment:
You might consider renaming some of the subaction methods to be the same as the key.  That could be a next step, but it seems a bit weird to have a bunch of actions called "observations_of_"...

Reply:
No problem. Sounds good - I had imagined at some point to go back through and try making the index params more consistent. thanks!